package.json

```diff
@@ -11,7 +11,7 @@
   },
   "devDependencies": {
     "jest": "^21.2.1",
-    "nuxt": "^1.2.0",
+    "nuxt": "^1.4.0",
```

Comment:
Also bump in the README…

Reply:
I will remove it from the README. We still have it in this file. *If* there are incompatibilities between versions it might be better to put the compatibility notes to the release/tag description.

Reply:
👍